website/addons/owncloud/tests/test_models.py

```diff
@@ -0,0 +1,88 @@
+from nose.tools import *  # noqa
+import mock
+
+from tests.base import get_default_metaschema
+from framework.auth.decorators import Auth
+
+from website.addons.base.testing import models
+
+from website.addons.owncloud.model import AddonOwnCloudNodeSettings
+from website.addons.owncloud.tests.factories import (
+    OwnCloudAccountFactory, OwnCloudNodeSettingsFactory,
+    OwnCloudUserSettingsFactory
+)
+from website.addons.owncloud.tests import utils
+
+class TestNodeSettings(models.OAuthAddonNodeSettingsTestSuiteMixin, utils.OwnCloudAddonTestCase):
+
+    short_name='owncloud'
+    ExternalAccountFactory = OwnCloudAccountFactory
+    NodeSettingsFactory = OwnCloudNodeSettingsFactory
+    NodeSettingsClass = AddonOwnCloudNodeSettings
+    UserSettingsFactory = OwnCloudUserSettingsFactory
+
+
+    def _node_settings_class_kwargs(self, node, user_settings):
+        return {
+            'user_settings': self.user_settings,
+            'folder': '/Documents',
+            'owner': self.node,
+            'node': self.node
+        }
+
+    def setUp(self):
+        super(TestNodeSettings, self).setUp()
+        self.set_node_settings(self.node_settings)
+
+    def test_create_log(self):
```

Comment:
Please put overridden tests beneath an `## Overrides` comment, with any non-overriden but newly-defined tests above.

Also this override appears superfluous -- the only difference is that `path` has been renamed to `filename`, which should be changed